Code/Framework/AzToolsFramework/AzToolsFramework/API/PythonLoader.cpp

```diff
@@ -40,9 +41,12 @@ namespace AzToolsFramework::EmbeddedPython
         AZ::IO::PathView libPythonName = AZ::IO::PathView(PYTHON_SHARED_LIBRARY_PATH).Filename();
         AZ::IO::FixedMaxPath pythonVenvLibPath = pythonVenvPath / "lib" / libPythonName;
 
-        m_embeddedLibPythonModuleHandle = AZ::DynamicModuleHandle::Create(pythonVenvLibPath.StringAsPosix().c_str(), false);
-        bool loadResult = m_embeddedLibPythonModuleHandle->Load(false, true);
-        AZ_Error("PythonLoader", loadResult, "Failed to load %s.\n", libPythonName.StringAsPosix().c_str());
+        if (AZ::IO::SystemFile::Exists(pythonVenvLibPath.StringAsPosix().c_str()))
```

Comment:
StringAsPosix() is not needed when passing the path into the Exist call.
The path would contain forward slashes on non-Windows platforms already and on Windows a backslash path separator works without issue.
```suggestion
        if (AZ::IO::SystemFile::Exists(pythonVenvLibPath.c_str()))
```